dashboard/modules/job/job_manager.py

```diff
@@ -517,15 +520,18 @@ async def submit_job(
         # returns immediately and we can catch errors with the actor starting
         # up.
         try:
+            resources = None
+            if _driver_on_current_node:
```

Comment:
You shouldn't do this. Do it like here: https://docs.ray.io/en/latest/ray-core/tasks/scheduling.html#scheduling-strategy